graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/CqlDirectives.java

```diff
@@ -485,6 +485,7 @@
   public static final String WHERE_OR_IF_OR_INCREMENT_FIELD = "field";
   public static final String WHERE_OR_IF_PREDICATE = "predicate";
   public static final String INCREMENT_PREPEND = "prepend";
+  public static final String TIMESTAMP = "cql_timestamp";
```

Comment:
Might be good to be more specific and call this a `WRITE_TIMESTAMP` and `cql_write_timestamp` to avoid confusion later if there's some other timestamp that gets added.

Reply:
the naming is unified with other directives:
- `cql_increment`
- `cql_ttl`
- `cql_timestamp`
So I am not sure if that necessary. Do you have a particular use case in mind?
(@olim7t wdyt about making this directive "longer": `cql_write_timestamp`?)

Reply:
I think `cql_timestamp` is specific enough. There is no other timestamp that can be sent with a request. As for reading back timestamps, this is done with the `writetime` CQL function; we don't have a way to do that yet, but I think we would do it with a way to provide a formula for an entity field, not via a dedicated directive.

Reply:
For the latter case, even if we would want to use a directive, I don't see a clear problem with using the same directive name as for request.